### PR TITLE
Add scoring

### PR DESCRIPTION
**Pull request by jonchan51**
Currently, scoring is just implemented as accuracy. We can think about whether we want some actual raw number in the future.
The way it works right now is that every 0.5s (by setInterval), it compares the currently pressed keys with the keys it expects to be pressed in the JSON at that exact moment. If equal, it marks that check as correct, and increments the total correct checks. Accuracy is determined by total correct checks / total checks. 

Need to test on different browsers, not sure if it's just me, on firefox it lags quite a bit towards the end, might just be my comp though

### CODE REVIEW

File: src/components/Game/GameView.tsx [hunk 1]
```diff
@@ -141,11 +184,13 @@ const GameView: React.FC<Props> = ({
   const didPlayNote = (note: number, playedBy: number) => {
     // TODO: update score
     console.log('Play', Tone.now() - delayedStartTime);
+    pressedNotes.current.add(note);
```
Comment (Na-Nazhou) [inline src/components/Game/GameView.tsx:187]: this callback is trigged for both my notes, and my friend notes, in case u are not aware

Comment (Na-Nazhou) [reply src/components/Game/GameView.tsx:187]: so u might want to store the playerId info as well

Comment (jonchan51) [reply src/components/Game/GameView.tsx:187]: ah yeah i didn't know that. Changed to only consider my own notes.

**Review by Na-Nazhou — APPROVED** [review]

**Comment by Na-Nazhou** [comment]
Close #37

**Pull request merged**